### DISCUSSION TITLE
Deployment Issue

### DISCUSSION THREAD

**Discussion by srabyanamrod** [vercel/vercel#10731] [Help]
### Question

We get an error like the one below. When we want to upload the latest version of our codes, can you help us solve the problem? Why might this problem occur?

[https://github.com/srabyanamrod/Blockchain/blob/master/vercel_%C4%B1ssue.jpeg](url)

**Comment by alejogamboa** [comment]
We are getting the same problem since yesterday execuing vercel cli to upload the latest version, I currenly have two projects, one in astro and the other one in nuxt, deploy fail in both cases for the same reason.

<img width="1136" alt="Captura de pantalla 2023-10-19 a la(s) 10 54 02 a  m" src="https://github.com/vercel/vercel/assets/5664550/39fc8623-2799-4c3f-8cf8-c17758d1a5cc">

**Reply by alejogamboa** [reply]
Solved, in my case I try previously run the project with bun I clean the proyect and rebuild it again and that's all
